website/docs/docs/core/connect-data-platform/gaussdbdws-setup.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Gaussdb(DWS) setup"
+description: "Read this guide to learn about the Gaussdb(DWS) warehouse setup in dbt."
+id: "gaussdbdws-setup"
+meta:
+  maintained_by: dbt Labs
+  authors: 'core dbt maintainers'
+  github_repo: 'n/a'
+  pypi_package: 'dbt-gaussdbdws'
+  min_core_version: 'v0.4.0'
+  cloud_support: Not supported
+  min_supported_version: 'n/a'
+  slack_channel_name: 'n/a'
+  slack_channel_link: 'n/a'
+  platform_name: 'Gaussdb(DWS)'
+  config_page: '/reference/resource-configs/gaussdbdws-configs'
+---
+
+<Snippet path="warehouse-setups-cloud-callout" />
+
+import SetUpPages from '/snippets/_setup-pages-intro.md';
+
+<SetUpPages meta={frontMatter.meta} />
+
+
+## Profile Configuration
+
+Gaussdb(DWS) targets should be set up using the following configuration in your `profiles.yml` file.
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+company-name:
+  target: dev
+  outputs:
+    dev:
+      type: gaussdbdws
+      host: [hostname]
+      user: [username]
+      password: [password]
+      port: [port]
+      dbname: [database name] # or database instead of dbname
+      schema: [dbt schema]
+      threads: [optional, 1 or more]
+      [keepalives_idle](#keepalives_idle): 0 # default 0, indicating the system default. See below
+      connect_timeout: 10 # default 10 seconds
+      [retries](#retries): 1  # default 1 retry on error/timeout when opening connections
```

Comment:
```suggestion
      retries: 1  # default 1 retry on error/timeout when opening connections
```

Reply:
Could you please take a look at your profiles.yml sample file? I think your fields are coming out a bit funny here. Thank you!

Reply:
like this:
jaffle_shop:
  target: dev_dws
  outputs:
    dev_dws:
      type: gaussdbdws
      host: 121.xxx.xxx.199
      user: xxx
      password: xxx@123
      port: 8000
      dbname: gaussdb
      schema: xxx
      threads: 1